datatypes/src/spatial_reference.rs

```diff
@@ -69,6 +70,19 @@ impl SpatialReference {
             }
         }
     }
+
+    pub fn area_of_use(self) -> Result<BoundingBox2D> {
```

Comment:
maybe add a note that this will be in lat/lon. And maybe add a method for a projected area of use?

Reply:
7c3dcc44c62f4ead9564512f5d7b4f05479b4d43